mcs/class/referencesource/mscorlib/system/threading/threadpool.cs

```diff
@@ -1145,10 +1149,11 @@ private bool ValidHandle()
 
     [System.Runtime.InteropServices.ComVisible(true)]
 #if FEATURE_REMOTING
-    public sealed class RegisteredWaitHandle : MarshalByRefObject {
+    public sealed class RegisteredWaitHandle : MarshalByRefObject
 #else // FEATURE_REMOTING
-    public sealed class RegisteredWaitHandle {
+    public sealed class RegisteredWaitHandle
 #endif // FEATURE_REMOTING
+    {
```

Comment:
anything under `mcs/class/referencesource` flows from .net framework referencesource and better not to changes it to make the merges less painful